doc/api/fs.md

```diff
@@ -1583,6 +1583,29 @@ fs.open('<directory>', 'a+', (err, fd) => {
 });
 ```
 
+Some characters are reserved under Windows which documented by
+[Naming Files, Paths, and Namespaces][].
+
++ `<` (less than)
++ `>` (greater than)
++ `:` (colon)
++ `"` (double quote)
++ `/` (forward slash)
++ `\` (backslash)
++ `|` (vertical bar or pipe)
++ `?` (question mark)
++ `*` (asterisk)
+
+Node.js would get an error while opening any file path containes characters
```

Comment:
If you drop the list, this sentence could be combined with the previous one. Also containes -> contains.